aws/aws.go

```diff
@@ -315,6 +315,11 @@ func (b *Backend) AckMessage(ctx context.Context, providerMetadata interface{})
 	return err
 }
 
+// GetSubName returns the subscription the message is from
+func (b *Backend) GetSubName(msg *hedwig.Message) string {
+	panic("not implemented yet for aws")
```

Comment:
@maroux we can add this if the implementation is easy for aws not sure if you know off the top of your head